src/rendering/RenderUtil.cc

```diff
@@ -2161,5 +2161,21 @@ void RenderUtil::ViewCollisions(const Entity &_entity)
 
     this->dataPtr->viewingCollisions[colEntity] = showCol;
     colVisual->SetVisible(showCol);
+
+    if (showCol)
+    {
+      // turn off wireboxes for collision entity except for most recent
+      // selected entity (it is still highlighted in entity tree)
```

Comment:
How about also showing collision for all selected entities? There may still be more than one selected. I tried with `log_record_dbl_pendulum.sdf`:

![coll](https://user-images.githubusercontent.com/5751272/112244956-454bef80-8c0d-11eb-866b-c8e64c96f647.gif)

Reply:
Since collision entities aren't created until after the first "View > Collisions", `RenderUtil::SetSelectedEntities` does nothing (does not add the collision entity to the list since it's not created yet) so there is no way of getting selected entities until after collisions are first created (toggled on). To avoid user confusion, I went ahead and turned off the bounding box for all selected collisions when toggling on: 4dfe6d32614474aad1819a532d11b0978294e594

Reply:
Nice! This behaviour is more consistent! :+1: